Final Project Proposal

christopherliu830 opened the pull request.

christopherliu830 committed:
Final Project

christopherliu830 committed:
test

christopherliu830 committed:
Add 'Game/' from commit '11ab32166c9f90d97bd43d6b9a030698533b00ca'

git-subtree-dir: Game
git-subtree-mainline: 18fffed08b0d8fb94c2f8aa97d5b49f3a447d22c
git-subtree-split: 11ab32166c9f90d97bd43d6b9a030698533b00ca

christopherliu830 committed:
Added .txt proposal

leighannehsu commented on Proposal.txt:
```diff
@@ -0,0 +1,28 @@
+	f.	Help – gives all available commands
+	g.	yell – call something, doesn’t do anything most of the time
+	h.	Hint – Maybe implement suggested commands
+	i.	put – try to put an item in something else
```
Are you intending to also house the summaries of each room in a function?

You should maybe come up with a loose story to work with before you continue.  You will need to flesh that out in your readme.txt that you submit with the final project.

christopherliu830 committed:
added rooms, goto,

input sanitization
moveto function

christopherliu830 committed:
added and moved functions, added a little story

added look
note
take
other simplifications
A bit of story

christopherliu830 committed:
Finished game

cleaned up output (newlines)
polished up rooms
Added end game (shack)
notes
Pretty unnecessary quit room
todo: documentation, comments

christopherliu830 committed:
forgot the a.out

christopherliu830 committed:
Revert "forgot the a.out"

This reverts commit f8ce207a53898289de6d37cbd41f45c11a65d39d.

christopherliu830 committed:
Reorganized files

christopherliu830 committed:
fixed recompile again

christopherliu830 committed:
Fixed memory leaks

still room for optimization

christopherliu830 committed:
Polish, optimization, actually runs now

moved the structs
got rid of unnecessary variables
changed park

christopherliu830 committed:
final comments

christopherliu830 committed:
updated a.out

christopherliu830 committed:
updated readme

christopherliu830 committed:
deleted

christopherliu830 committed:
Final update to readme
